src/main/java/me/totalfreedom/totalfreedommod/command/Command_disguisetoggle.java

```diff
@@ -22,7 +21,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             return true;
         }
 
-        FUtil.adminAction(sender.getName(), (DisallowedDisguises.disabled ? "Enabling" : "Disabling") + " Disguises", false);
+        FUtil.adminAction(sender.getName(), (!plugin.ldb.isDisguisesEnabled() ? "Enabling" : "Disabling") + " Disguises", false);
```

Comment:
@OxLemonxO, the code is correct. If the disguises are disabled, it would need to say Enabling not Disabling.